kong.conf.default

```diff
@@ -658,12 +658,16 @@
                                  # used for verifying Kong's database
                                  # connections, when `pg_ssl_verify` or
                                  # `cassandra_ssl_verify` are enabled.
+#
+# See https://github.com/openresty/lua-nginx-module#lua_ssl_trusted_certificate
 
 #lua_ssl_verify_depth = 1        # Sets the verification depth in the server
                                  # certificates chain used by Lua cosockets,
                                  # set by `lua_ssl_trusted_certificate`.
                                  # This includes the certificates configured
                                  # for Kong's database connections.
+#
+# See https://github.com/openresty/lua-nginx-module#lua_ssl_verify_depth
```

Comment:
please indent these — the other long URLs in this file are all indented (even if they go beyond 80 characters)